support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -122,22 +122,22 @@ const testDetailsPromo = [
 		frequency: 'Monthly',
 		promoCode: 'E2E_TEST_SPLUS_MONTHLY',
 		expectedCardHeading: 'All-access digital',
-		expectedPromoText: '£8/month for 6 months, then £10/month',
-		expectedCheckoutTotalText: 'Was £10, now £8/month',
-		expectedCheckoutButtonText: 'Pay £8 per month',
+		expectedPromoText:
+			/£(\d|\.)+\/month for (\d|\.) months, then £(\d|\.)+\/month/,
```

Comment:
Possibly '(\d) months' in here?

Reply:
I've just left it to be consistent. Happy to change if you think it's unclear.